tmk_core/common/action_util.h

```diff
@@ -18,22 +18,45 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 #include <stdint.h>
+#include <stddef.h>
 #include "report.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-extern report_keyboard_t *keyboard_report;
+#if !defined(QMK_KEYS_PER_SCAN) && defined(REGISTER_MULTIPLE_KEYEVENTS_ENABLED)
+#    warning REGISTER_MULTIPLE_KEYEVENTS_ENABLED needs QMK_KEYS_PER_SCAN defined to have any effect, setting default of 4.
```

Comment:
```suggestion
```

Warning are treated as errors and stops the compilation (at least by default). If it can just define a decent default does it need to show a warning?

Reply:
Converted to `#pragma message`